### PR TITLE
Correct spec documentation for GitHubSource

### PR DESCRIPTION
**Pull request by johscheuer**
Correct the `serviceAccountName` description for the `GitHubSource` and fix a missing space.

### CODE REVIEW

File: eventing/README.md [hunk 1]
```diff
@@ -168,8 +168,8 @@ The GitHubSource fires a new event for selected
   [SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#secretkeyselector-v1-core)
   containing a GitHub secret token for configuring a GitHub webhook. One of this
   or `accessToken` must be set.
-- `serviceAccountName`: `string` The name of the ServiceAccount used to access
-  the `gcpCredsSecret`.
+- `serviceAccountName`: `string` The name of the ServiceAccount to run the
```
Comment (samodell) [inline eventing/README.md]:
```suggestion
- `serviceAccountName`: `string` The name of the ServiceAccount running as the
```

File: eventing/README.md [hunk 2]
```diff
@@ -168,8 +168,8 @@ The GitHubSource fires a new event for selected
...
+  container as.
```
Comment (samodell) [inline eventing/README.md]:
```suggestion
  container.
```

**Comment by johscheuer** [comment]
Thanks for your suggestion I just oriented at the other descriptions, e.g.: https://github.com/knative/docs/blob/master/eventing/README.md#containersource

**Comment by samodell** [comment]
@johscheuer 
> Thanks for your suggestion I just oriented at the other descriptions, e.g.: [https://github.com/knative/docs/blob/master/eventing/README.md#containerso](https://github.com/knative/docs/blob/master/eventing/README.md#containersource)

Ah, thanks for pointing out where that came from! Let's actually leave it unchanged for now. Sorry f or the back-and-forth.

**Comment by johscheuer** [comment]
Reverted the changed :)

**Pull request merged**